requirements/base.in

```diff
@@ -5,3 +5,6 @@ Django                                  # Web application framework
 openedx-events                          # Events API
 edx_django_utils
 edx_toggles
+
+# newrelic==8.2.0 has broken instrumentation for confluent-kafka. This constraint can be removed once this issue is fixed.
+newrelic<8.2.0
```

Comment:
Is this too late for the constraint? I wonder if we need to instead constrain in edx-platform and discovery, because of how we are bringing in this dependency. Not sure.

Reply:
This dependency we bring in normally. confluent_kafka is the one where it's weird. Or are you talking about something else?

Reply:
I thought so too but @jmbowman said otherwise. The original fix may just have been about generating docs. I wasn't totally clear on all the effects when I was working on it.

Reply:
Maybe @jmbowman is forgetting about this update: https://github.com/openedx/edx-cookiecutters/commit/f8f4c9c5162d4b4ee7def4f46ab8cd31071d93db, or maybe I am misunderstanding what it does.